secscanner2junit/vulnerability.py

```diff
@@ -105,19 +107,16 @@ def get_severity(self):
 class SastVulnerability(_Vulnerability):
```

Comment:
Maybe we can create `SecrectsVulnerability(_Vulnerability):` ?

Reply:
The secrets output is compatible with sast so I chose not to do this. I could subclass the SastVulnerability or introduce a new SastBaseVulnerability. And do the same for the SastLocation? It felt like adding duplication so I was on the fence about this. If you have a strong opinion on this I'm happy to make the change.

Reply:
it was based on https://gitlab.com/gitlab-org/security-products/security-report-schemas/-/tree/master/src

Both of this report are based on https://gitlab.com/gitlab-org/security-products/security-report-schemas/-/blob/master/src/sast-report-format.json

So maybe your idea is correct :)

We could model classes to be comatible with this schemas what will be useful to others?

@Urth What do you think?